Sources/ClientRuntime/Networking/AwsChunked/AWSChunkedStream.swift

```diff
@@ -130,10 +130,11 @@ extension AWSChunkedStream: Stream {
         // If the stream is closed or some other Task has already replenished the stream,
         // then this operation is not needed.  Exit.
         guard !outputStream.isClosed, outputStream.bufferCount == 0 else { return }
+
         // Process the first chunk and determine if there are more to send
         let hasMoreChunks = try await chunkedReader.processNextChunk()
         let currentChunkBodyIsEmpty = chunkedReader.getCurrentChunkBody().isEmpty
-//        try await Task.sleep(nanoseconds: UInt64(Double.random(in: 0.1...1.0) * 1_000_000_000.0))
+
```

Comment:
Sorry, this was mine, must have left this in by mistake.  (But at least I left it in commented)